tests/test_dynamic_boundary_conditions/data/hirds_output.txt

```diff
@@ -0,0 +1,151 @@
+HIRDS V4 Depth-Duration-Frequency Results
```

Comment:
great to see a test added.

Reply:
will request another review after I've finished adding tests :)